keysign/QRCode.py

```diff
@@ -317,5 +318,8 @@ def on_released(widget, event):
 if __name__ == '__main__':
     import sys
     logging.basicConfig(level=logging.DEBUG)
-    data = sys.argv[1]
+    if len(sys.argv) > 1:
+	    data = sys.argv[1]
+    else:
+            raise ValueError("Not Enough Arguments passed as data for the QR code encoding")
```

Comment:
Indented too much. 12 spaces instead of 8.

Reply:
@RyuzakiKK Done. Sorry about that.